package.json

```diff
@@ -20,10 +20,15 @@
   },
   "homepage": "https://github.com/dandevri/minor-wot#readme",
   "dependencies": {
+    "cookie-parser": "^1.4.3",
+    "dotenv": "^4.0.0",
     "ejs": "^2.5.6",
     "express": "^4.15.2",
     "lodash": "^4.17.4",
-    "node-fetch": "^1.6.3"
+    "node-fetch": "^1.6.3",
+    "querystring": "^0.2.0",
+    "request": "^2.81.0",
```

Comment:
looks like we are using two different `npm` modules to do an HTTP request. We should probably fix this.